sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/DecryptResult.cs

```diff
@@ -14,7 +14,7 @@ public class DecryptResult : IJsonDeserializable
         private const string PlaintextPropertyName = "value";
 
         /// <summary>
-        /// The <see cref="KeyBase.Id"/> of the <see cref="Key"/> used to decrypt
+        /// The <see cref="KeyProperties.Id"/> of the <see cref="Key"/> used to decrypt
         /// </summary>
         public string KeyId { get; internal set; }
```

Comment:
The usage of KeyId seems to be mixed in these classes.  Sometimes they are strings sometimes Uri.  They probably should be Uri across the whole library.  We don't need to fix it in the PR necessarily, but if not we should create an issue to track this.

Reply:
New issue #7760 opened.